### ISSUE TITLE
[Tooltip] ariaLabel read twice 

### ISSUE THREAD

**Issue by skarensmoll** [radix-ui/primitives#2988]
### Current Behavior

When Tooltip.Trigger's `aria-label` has the same content as `Tooltip.Content`. The voiceover reads it twice.

```
 <Tooltip.Root>
    <Tooltip.Trigger asChild>
      <IconButton aria-label="Add to library">
        <PlusIcon />
      </IconButton>
    </Tooltip.Trigger>

    <TooltipContent sideOffset={5}>
      Add to library
      <TooltipArrow />
    </TooltipContent>
  </Tooltip.Root>

```

<img width="784" alt="Screenshot 2024-07-02 at 13 32 10" src="https://github.com/radix-ui/primitives/assets/7997951/43fa469d-adc1-4aab-8796-fee0cb78d29c">

### Expected behavior

When `aria-label` and Content is the same, `aria-describedby` attribute should be removed, so only `aria-label` is read.

### Reproducible example

[[CodeSandbox Template](https://codesandbox.io/s/2r30e)](https://codesandbox.io/p/devbox/great-shockley-pl3qfh?file=%2FApp.jsx%3A1%2C1-88%2C1&workspaceId=64755a69-6ad6-43fc-a87e-648c3f009eed)

**Comment by daniellesuper** [comment]
pretty sure this is related to and the same issue as [Issue 3034](https://github.com/radix-ui/primitives/issues/3034)

**Comment by Watso196** [comment]
Additionally, when we want to relate a Tooltip's content to a Trigger control as the accessible label, we get duplicate read out there. It would be great to have this resolved and be able to use Tooltips as labels for icon buttons!

**Comment by aallred406** [comment]
In this case, it would be better to use `aria-labelledby` and point to the `TooltipContent`. That way you won't be labelling the button twice.
```
 <Tooltip.Root>
    <Tooltip.Trigger asChild>
      <IconButton aria-labelledby="button-label">
        <PlusIcon />
      </IconButton>
    </Tooltip.Trigger>

    <TooltipContent id="button-label" sideOffset={5}>
      Add to library
      <TooltipArrow />
    </TooltipContent>
  </Tooltip.Root>
```

**Comment by jeffdaley** [comment]
> In this case, it would be better to use `aria-labelledby` and point to the `TooltipContent`. That way you won't be labelling the button twice.
> 
> ```
>  <Tooltip.Root>
>     <Tooltip.Trigger asChild>
>       <IconButton aria-labelledby="button-label">
>         <PlusIcon />
>       </IconButton>
>     </Tooltip.Trigger>
> 
>     <TooltipContent id="button-label" sideOffset={5}>
>       Add to library
>       <TooltipArrow />
>     </TooltipContent>
>   </Tooltip.Root>
> ```

`aria-label` in this case allows the button to have a label without it being interacted with. If you use `described-by` you'll be pointing to a non-existent element until hover/focus. You don't want unlabelled buttons in the Accessibility Tree:

<img width="203" height="58" alt="Image" src="https://github.com/user-attachments/assets/828edf21-66a8-48dc-a771-3a261118d758" />

**Comment by chaance** [comment]
Duplicate of https://github.com/radix-ui/primitives/issues/3689
